label_studio/core/settings/base.py

```diff
@@ -400,6 +400,31 @@
 UPLOAD_DIR = 'upload'
 AVATAR_PATH = 'avatars'
 
+SUPPORTED_EXTENSIONS = set(
```

Comment:
There are no video formats in this list.
Here's Chrome supported formats: https://sites.google.com/a/chromium.org/dev/audio-video

Reply:
Added video extensions